src/QuickMarcEditor/utils.js

```diff
@@ -24,6 +29,50 @@ export const dehydrateMarcRecordResponse = marcRecordResponse => ({
   ],
 });
 
+const removeMarcRecordFieldContentForDuplication = marcRecord => {
+  const shouldRemoveFieldContent = (field) => {
+    if (!FIELD_TAGS_TO_REMOVE.includes(field.tag)) {
+      return false;
+    }
+
+    const [firstIndicator, secondIndicator] = (field.indicators || []);
+
+    if (field.tag === '999') {
+      if (firstIndicator === 'f' && secondIndicator === 'f') {
+        return true;
+      }
+
+      return false;
+    }
+
+    return true;
+  };
+
+  return {
+    ...marcRecord,
+    records: marcRecord.records.map(field => (shouldRemoveFieldContent(field)
```

Comment:
Wouldn't be better to wrap single argument into brackets too? (according to the https://github.com/airbnb/javascript#arrows--paren-wrap)
```suggestion
    records: marcRecord.records.map((field) => (shouldRemoveFieldContent(field)
```